tests/components/zeroconf/test_init.py

```diff
@@ -85,12 +91,16 @@ def mock_homekit_info(service_type, name):
 async def test_homekit_match_full(hass, mock_zeroconf):
     """Test configured options for a device are loaded via config entry."""
     with patch.dict(
-        zc_gen.ZEROCONF, {zeroconf.HOMEKIT_TYPE: ["homekit_controller"]}, clear=True
-    ), patch.object(hass.config_entries, "flow") as mock_config_flow, patch.object(
-        zeroconf, "ServiceBrowser", side_effect=service_update_mock
-    ) as mock_service_browser:
-        mock_zeroconf.get_service_info.side_effect = get_homekit_info_mock("BSB002")
-        assert await async_setup_component(hass, zeroconf.DOMAIN, {zeroconf.DOMAIN: {}})
+            zc_gen.ZEROCONF, {zeroconf.HOMEKIT_TYPE: ["homekit_controller"]},
+            clear=True), patch.object(
+                hass.config_entries, "flow") as mock_config_flow, patch.object(
+                    zeroconf,
+                    "ServiceBrowser",
+                    side_effect=service_update_mock) as mock_service_browser:
+        mock_zeroconf.get_service_info.side_effect = get_homekit_info_mock(
+            "BSB002")
+        assert await async_setup_component(hass, zeroconf.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)